From 56a0de5e9b3435eebeed37a4eb4675a0ffa8d95b Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:58:58 -0500
Subject: [redhat] kabi: Adding symbol __next_cpu_nr (lib/cpumask.c)

Message-id: <e090da1674f8e7f3a8d7b7934892b4216d98cc3e.1418903403.git.poros@redhat.com>
Patchwork-id: 102252
O-Subject: [PATCH RHEL7.1 06/48] kabi: Adding symbol __next_cpu_nr (lib/cpumask.c)
Bugzilla: 1160751
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160751
    https://bugzilla.redhat.com/show_bug.cgi?id=1160751

    Description:
    ============
    Called via for_each_cpu_mask_nr
    Used to itterate over cpus when pfn is not in the same memory node.
    Used in xpmem.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__next_cpu_nr
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64le/__next_cpu_nr
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__next_cpu_nr
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__next_cpu_nr

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__next_cpu_nr b/redhat/kabi/kabi-module/kabi_ppc64/__next_cpu_nr
new file mode 100644
index 0000000..80fcd54
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__next_cpu_nr
@@ -0,0 +1,2 @@
+#1-
+0x5289b437 __next_cpu_nr vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/__next_cpu_nr b/redhat/kabi/kabi-module/kabi_ppc64le/__next_cpu_nr
new file mode 100644
index 0000000..80fcd54
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/__next_cpu_nr
@@ -0,0 +1,2 @@
+#1-
+0x5289b437 __next_cpu_nr vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__next_cpu_nr b/redhat/kabi/kabi-module/kabi_s390x/__next_cpu_nr
new file mode 100644
index 0000000..47f7957
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__next_cpu_nr
@@ -0,0 +1,2 @@
+#1-
+0xa5e4f65c __next_cpu_nr vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__next_cpu_nr b/redhat/kabi/kabi-module/kabi_x86_64/__next_cpu_nr
new file mode 100644
index 0000000..a213aae
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__next_cpu_nr
@@ -0,0 +1,2 @@
+#1-
+0xbba6f82e __next_cpu_nr vmlinux EXPORT_SYMBOL
-- 
1.7.1